starter-code/routes/authController.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const bcrypt= require('bcrypt');
+const authController = express.Router();
+
+const User = require('../models/user');
+const bcryptSalt = 10;
+
+authController.get('/signup', (req, res, next) => {
+    res.render('auth/signup')
+})
+
+authController.post('/signup', (req, res, next) => {
+    var username = req.body.username;
+    var name = req.body.name;
+    var email = req.body.email;
+    var password = req.body.password;
```

Comment:
Pour info, these four lines can be written shortly like this:

var { username, name, email, password } = req.body

It's a relatively new syntax that many people use today. We'll see it more later :)

Reply:
Thanks you for the info !

Reply:
Glad I lurked in here, me likey

Reply:
A whole new wooooorld